website/app/components/dynamic-template.js

```diff
@@ -25,9 +25,9 @@ export default Component.extend({
     this.templateMap = templateMap;
   },
 
-  componentName: computed('templateString', 'componentId', function() {
+  componentName: computed('templateString', 'componentId', function () {
```

Comment:
Prettier format on save turned on in vscode. I touched the file to add performance logging and then removed it.  I left them in since they seemed to be reasonable lint cleanups.

Reply:
Ahh. I see what's happening now. For some reason we have prettier disabled at the top of the file (and a few others that were part of that initial scaffolding) which is why this hasn't come up before/isn't failing the lining checks. Sort of puzzling why your editor ignored that and linted the file. It would be cleaner to extract that into a separate commit/PR and we could turn on linting for that file and a few others moving forward.

Reply:
makes sense, for sure, will do!